web/app/js/components/Sidebar.jsx

```diff
@@ -18,7 +19,13 @@ class Sidebar extends React.Component {
     this.loadFromServer = this.loadFromServer.bind(this);
     this.handleApiError = this.handleApiError.bind(this);
 
-    this.state = {
+    this.state = this.getInitialState();
+  }
+
+  getInitialState() {
+    return {
+      pollingInterval: 12000,
+      maxNsItemsToShow: 8,
```

Comment:
is there some way to indicate to the user that we're truncating the list at 8? perhaps ellipsis somewhere, or a count of the total namespaces?

Reply:
ooh, yeah, maybe a count would be good